src/viewer/__snapshots__/PdfJsViewer.spec.jsx.snap

```diff
@@ -0,0 +1,7 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`PDFViewer with a pdf that does not load should show a fallback 1`] = `
```

Comment:
maybe shallow rendering is not the best here?

Reply:
Mmmh I'll check this, I really don't want to render the pdf in a test, but maybe just of the error it would work.

Reply:
I guess you could mock `Document` component but whatever, it will be covered by e2e tests I guess !